README.md

```diff
@@ -160,6 +247,114 @@ of how some key tasks can be carried out using Valet:
 - [NPM](https://docs.npmjs.com/cli) commands may be executed by using `npm <command>`.
 - [Artisan](https://laravel.com/docs/10.x/artisan) commands may be executed by using `php artisan <command>`.
 
+
+### Local development setup using docker compose:
+1. Install docker according to your platform instructions found [here](https://docs.docker.com/get-docker/).
+2. Clone the repository:
+
+    ```bash
+    git clone https://github.com/accessibility-exchange/platform.git
+    cd platform
+    ```
+
+3. Create a `.env` file from the included example file:
+
+    ```bash
+    cp .env.local.example .env
+    ```
+   
+    Then, change the `APP_ENV` value to `local`:
+
+    ```dotenv
+    APP_ENV=local
+    ```
+
+4. Generate an encryption key for [CipherSweet](https://github.com/spatie/laravel-ciphersweet):
+
+    ```bash
+    docker run --rm -it alpine apk add openssl && openssl rand -hex 32
+    ```
+
+    Add it to your `.env` file:
+    
+    ```dotenv
+    CIPHERSWEET_KEY="<your key>"
+
+5. Build you application container that will be used to generate keys:  
+
+    ```bash
+    docker compose -f docker-compose.local.yml build platform.test
+    ```
+    ```
```

Comment:
Also I couldn't get past this step as it kept failing with an error about composer install related to filament. I deleted the images and volumes from Docker and also pruned the cache, but kept hitting this issue.

Error:

```bash
 => ERROR [platform.test stage-0 23/25] RUN composer install                                                                                                                                                      0.8s
------
 > [platform.test stage-0 23/25] RUN composer install:
0.571 Composer plugins have been disabled for safety in this non-interactive session. Set COMPOSER_ALLOW_SUPERUSER=1 if you want to allow plugins to run as root/super user.
0.571 Do not run Composer as root/super user! See https://getcomposer.org/root for details
0.684 Installing dependencies from lock file (including require-dev)
0.691 Verifying lock file contents can be installed on current platform.
0.760 Your lock file does not contain a compatible set of packages. Please run composer update.
0.760 
0.760   Problem 1
0.760     - filament/support is locked to version v3.0.74 and an update of this package was not requested.
0.760     - filament/support v3.0.74 requires ext-intl * -> it is missing from your system. Install or enable PHP's intl extension.
0.760   Problem 2
0.760     - filament/support v3.0.74 requires ext-intl * -> it is missing from your system. Install or enable PHP's intl extension.
0.760     - filament/widgets v3.0.74 requires filament/support v3.0.74 -> satisfiable by filament/support[v3.0.74].
0.760     - filament/widgets is locked to version v3.0.74 and an update of this package was not requested.
0.760 
0.760 To enable extensions, verify that they are enabled in your .ini files:
0.760     - /usr/local/etc/php/conf.d/docker-fpm.ini
0.760     - /usr/local/etc/php/conf.d/docker-php-ext-apcu.ini
0.760     - /usr/local/etc/php/conf.d/docker-php-ext-bcmath.ini
0.760     - /usr/local/etc/php/conf.d/docker-php-ext-bz2.ini
0.760     - /usr/local/etc/php/conf.d/docker-php-ext-exif.ini
0.760     - /usr/local/etc/php/conf.d/docker-php-ext-gd.ini
0.760     - /usr/local/etc/php/conf.d/docker-php-ext-imagick.ini
0.760     - /usr/local/etc/php/conf.d/docker-php-ext-opcache.ini
0.760     - /usr/local/etc/php/conf.d/docker-php-ext-pcntl.ini
0.760     - /usr/local/etc/php/conf.d/docker-php-ext-pdo_mysql.ini
0.760     - /usr/local/etc/php/conf.d/docker-php-ext-redis.ini
0.760     - /usr/local/etc/php/conf.d/docker-php-ext-soap.ini
0.760     - /usr/local/etc/php/conf.d/docker-php-ext-sodium.ini
0.760     - /usr/local/etc/php/conf.d/docker-php-ext-zip.ini
0.760 You can also run `php --ini` in a terminal to see which files are used by PHP in CLI mode.
0.760 Alternatively, you can run Composer with `--ignore-platform-req=ext-intl` to temporarily ignore these required extensions.
------
failed to solve: process "/bin/sh -c composer install" did not complete successfully: exit code: 2 
```

Reply:
> Also I couldn't get past this step as it kept failing with an error about composer install related to filament. I deleted the images and volumes from Docker and also pruned the cache, but kept hitting this issue.

This is actually an existing issue and has been filed as #1941

Reply:
> > Also I couldn't get past this step as it kept failing with an error about composer install related to filament. I deleted the images and volumes from Docker and also pruned the cache, but kept hitting this issue.
> 
> This is actually an existing issue and has been filed as #1941

This issue was addressed in #1942

Reply:
I ran into the same. It's an easy fix that I can include in this PR. It will fix kube builds as well since they share the same build.
We also have to include the livewire fix so that you can successfully build.

Reply:
I saw that both those were already merged into dev so I merged the latest dev branch into this PR. This should resolve these issues.

Reply:
There is still an extra set of back ticks here.